### PR TITLE
[ReportsConfig] Updated IModels Client

### PR DESCRIPTION
**Pull request by Guillar1**

### CODE REVIEW

File: change/@itwin-reports-config-widget-react-08145948-89dc-4064-a856-f17f7fbe3c64.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (MindaugasButkus) [inline change/@itwin-reports-config-widget-react-08145948-89dc-4064-a856-f17f7fbe3c64.json:2]: Is it a patch if major version changed of a dependency?

Comment (Guillar1) [reply change/@itwin-reports-config-widget-react-08145948-89dc-4064-a856-f17f7fbe3c64.json:2]: There is no introduction of new features or breaking changes. I believe a patch is appropriate.

Comment (MindaugasButkus) [reply change/@itwin-reports-config-widget-react-08145948-89dc-4064-a856-f17f7fbe3c64.json:2]: @aruniverse  ?

Comment (MindaugasButkus) [reply change/@itwin-reports-config-widget-react-08145948-89dc-4064-a856-f17f7fbe3c64.json:2]: Pre-release (`0.y.z`) require bumping `y` (minor) on breaking changes. Bumping the imodels-client major version might be a breaking change/

Comment (Guillar1) [reply change/@itwin-reports-config-widget-react-08145948-89dc-4064-a856-f17f7fbe3c64.json]: Changed it to minor.

**Review by MindaugasButkus — APPROVED** [review]

**Pull request merged**